.github/workflows /pr-open.yml

```diff
@@ -0,0 +1,45 @@
+name: Build Pull request
+on:
+  pull_request:
```

Comment:
looks like you need to mention branches?

Reply:
Seems its decided to work! :O

Reply:
Removed as no longer needed ✅